rdrf/rdrf/helpers/utils.py

```diff
@@ -861,6 +861,13 @@ def validate_abbreviated_name(value):
         raise ValidationError(_('Abbreviated name contains invalid characters. Accepted characters: Alphanumeric, spaces and dashes.'))
 
 
+def validate_file_extension_format(value):
+    if re.match(r'^\.([^\.]*)', value) is None:
```

Comment:
This will allow `.` as an extension. How about the following?

```regex
^\.[^\.]+$
```

Reply:
Updated to `^\.[^.]+$`. Pycharm told me the character escaping was unnecessary in the square brackets.